### DISCUSSION TITLE
skywalking ui：Time filtering does not match the data date

### DISCUSSION THREAD

**Discussion by zhangzhaogit** [apache/skywalking#7863] [Q&A]
version：oap、ui  8.7.0es7
skywalking ui：Time filtering does not match the data date
![1](https://user-images.githubusercontent.com/50043496/136227298-08ae1d8e-10b7-4854-be77-a180f43d23f0.png)
![2](https://user-images.githubusercontent.com/50043496/136227592-3d9bd9c0-d4ec-455c-b1f1-ae1ec30b49db.png)

**Comment by wu-sheng** [comment]
Please check the grapgql condition with considering time zone you selected.
I can't reproduce this on the demo env.
